packages/web-components/src/components/rux-calendar/rux-calendar.tsx

```diff
@@ -0,0 +1,547 @@
+import { Component, Host, h, Prop, Watch, Element, State } from '@stencil/core'
+import { getDay, getDaysInMonth, lastDayOfMonth } from 'date-fns'
+import { utcToZonedTime } from 'date-fns-tz'
+
+type MonthMap = {
+    [key: number]: string
+}
+
+const monthMap: MonthMap = {
+    1: 'January',
+    2: 'February',
+    3: 'March',
+    4: 'April',
+    5: 'May',
+    6: 'June',
+    7: 'July',
+    8: 'August',
+    9: 'September',
+    10: 'October',
+    11: 'November',
+    12: 'December',
+}
+
+@Component({
+    tag: 'rux-calendar',
+    styleUrl: 'rux-calendar.scss',
+    shadow: true,
+})
+export class RuxCalendar {
+    @Element() el!: HTMLRuxCalendarElement
+
+    /**
+     * Max date that the calendar will go to. Needs to be a valid date string.
+     */
+    @Prop() max?: string
+
+    /**
+     * Min date that the calendar will go to. Needs to be a valid date string.
+     */
+    @Prop() min?: string
+
+    /**
+     * Option to give the calendar a specfic month/year
+     */
+    @Prop({ attribute: 'date-in' }) dateIn?: string | number
```

Comment:
I think this should reflect so you can see the date-in attribute change in the DOM.